If-CI-checks-fail-on-your-PR.md

```diff
@@ -71,13 +71,12 @@ If you see that a test is failing, there are two possibilities: your code could
 
   This message is based on a list of known flakes maintained by the Automated QA team. Note however that this list may be incomplete, so a test could be flaking even if this message says `false`.
 
-If your code is wrong, then you'll need to fix it just as you would [respond to reviewer comments](https://github.com/oppia/oppia/wiki/Make-a-pull-request#step-5-address-review-comments-until-all-reviewers-approve). You may also want to review the documentation on our various CI checks to help you debug:
+If your code is wrong, then you'll need to fix it just as you would [respond to reviewer comments](https://github.com/oppia/oppia/wiki/Make-a-pull-request#step-5-address-review-comments-until-all-reviewers-approve). You may also want to review the documentation on our various CI checks to help you debug.
```

Comment:
Should this be a colon? Otherwise the bullet points below are orphaned.

Also the next couple of links aren't really about CI (e.g. the tests link is more general) -- should we link to something that's more specific to CI? Ideally readers would be able to easily map what they see in the CI checks to the wiki  page to look at, and for newcomers, having just these 2 bullet points might make that hard to do.

Reply:
done. At the top of the page I walk readers through identifying whether whether a CI failure is due to a failing test or lint check, and since our CI job names distinguish between the kinds of tests and lint checks, I think it should be pretty clear which link readers need to click on